fellow-submissions/peter-rose/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css" type="text/css" />
+</head>
+<body>
+  <div class="container-wrapper">
+    <main class="container">
+      <header>
+        <h4>Quiz'n You, my guy</h4>
+      </header>
+      
+      <section class="q1">
+      <h4>Info Please</h4>
+      <div style="float:left">
+    <label for="name">Name</label>
+    <input id="name" type="text" value="" name="name">
+  </div>
+
+  <div style="float:left;">
+    <label for="email">Email</label>
+    <input id="email" type="text" value="" name="email">
+  </div>
+
+
+      </section>
+      
+      
+      
+      <section class="q2">
+        <div>
+          <h4> Name this anime;<p>This anime is about some dude forcing a kid to eat his hair.</p></h4>
+
+        </div>
+      <input type="radio" name="" value="always">Made In Abyss
+      <input type="radio" name="" value="never">Vinland SAGA
+      <input type="radio" name="" value="costChange">My Hero Acadamia
+      </section>
+      
+                  <section class="q3">
+        <h5>Tell us about yaself</h5>
+        
+        <textarea placeholder="'Hi, my name is...'.........You get the idea"></textarea>
+
+      </section>
+      
+      <section class="q4">
```

Comment:
You're using `<section>`s for grouping and `<div>`s for headers. I want you to use `<fieldset>s` and `<legend>`s instead.